public/emails/new_user_invite.html

```diff
@@ -267,7 +267,7 @@ <h4 class="center" style="color: #222222; font-family: 'Open Sans', 'Helvetica N
 											<center style="width: 100%; min-width: 580px;">
 												<p style="font-size: 12px; color: #999999; font-family: 'Open Sans', 'Helvetica Neue', 'Helvetica', Helvetica, Arial, sans-serif; font-weight: normal; line-height: 19px; -webkit-font-smoothing: antialiased; -webkit-text-size-adjust: none; margin: 0 0 10px; padding: 0;" align="center">
 													Sent by <a href="{{.AppUrl}}" style="color: #E67612; text-decoration: none;">Grafana v{{.BuildVersion}}</a>
-													<br />© 2019 Grafana Labs
+													<br />© 2019 SANgraphs
```

Comment:
```suggestion
													<br />© 2020 SANgraphs
```

Reply:
Ideally there will be a variable for this, but I am not sure there is one.

Reply:
I think the next commit fixes it